web-client/src/views/FileHandlingHelpers/pdfValidation.ts

```diff
@@ -21,6 +26,63 @@ export const validatePdf = ({
 }: {
   file: File;
 }): Promise<FileValidationResponse> => {
+  const isBrowserSupported = (): boolean => {
```

Comment:
I love the idea here, but I don't like this solution.

- We are relying on the User Agent string and regex parsing, which is can be thorny and brittle.
- Some older browsers might, in fact, work, even if we think they shouldn't. In other words, we are aborting early before confirming there is actually a problem.

I have two suggestions:

- Use feature detection rather than user agent parsing. See the commented out code in https://github.com/ustaxcourt/ef-cms/blob/staging/shared/src/business/utilities/pdfs/getPdfJs.ts for how we did this before.
- Rather than aborting early, let the user at least try to do the thing. On failure, check if the browser is old via that feature detection. If so, message accordingly.